### DISCUSSION TITLE
Audio Issues

### DISCUSSION THREAD

**Discussion by it-dalilk** [deepgram/community#1014] [General help]
Hi,

I am using the pre-recorded audio API:
https://api.deepgram.com/v1/listen?smart_format=true&model=nova-2-conversationalai&language=en-US.
I send the audio file in the body as body: mp3Blob. However, my clients are encountering multiple issues, including:

Slow upload errors: This frustrates users and sometimes causes them to leave the page.
Empty transcripts: The API returns no content for certain recordings.
Bad Request errors: The API responds with errors like "failed to process audio: corrupt or unsupported data."
Here are examples of the error messages (with request IDs):

{"err_code":"Bad Request","err_msg":"Bad Request: failed to process audio: corrupt or unsupported data","request_id":"f78c45cd-9efd-41e1-bbed-714fa8e88174"}
{"err_code":"SLOW_UPLOAD","err_msg":"Request upload timeout.","request_id":"7b993e42-cac9-4620-86f3-28e47395bcde"}
{"err_code":"Bad Request","err_msg":"Bad Request: failed to process audio: corrupt or unsupported data","request_id":"3e00c5bf-5714-4d8b-82f7-57275d3d5014"}
I am using the JavaScript MediaRecorder library for recording audio on the client side.

How can I resolve these issues to ensure smooth and error-free processing for my clients?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Reply by it-dalilk** [reply]
Hi, 

I tried to connect it but I'm getting an error from Github:
![image](https://github.com/user-attachments/assets/6d954300-c26c-4107-b065-840e784e38c2)

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The deepgram product you are using (e.g Speech to Text, Agent API)

**Reply by it-dalilk** [reply]
I'm using Speech to Text

**Comment by it-dalilk** [comment]
@team-deepgram What's the solution for the slow API reponse?

**Comment by deadcoder0904** [comment]
@it-dalilk did you find a fix for this? i'm facing the same thing. nearly, 50% of requests give the same error for me.
